### ISSUE TITLE
Modify default ffmpeg string

### ISSUE THREAD

**Issue by bruno-f-cruz** [AllenNeuralDynamics/Aind.Behavior.Services#100]
@galenlynch Feel free to add the two here. Thanks!

**Comment by galenlynch** [comment]
Here is the string that I used:
```
ffmpeg -y -v verbose -colorspace rgb -color_primaries bt709 -color_trc linear -i <input> -vf "scale=out_color_matrix=bt709:out_range=full" -c:v hevc_nvenc -pix_fmt p010le -color_range full -colorspace bt709 -color_trc linear -fps_mode passthrough -tune hq -preset p4 -rc vbr -cq 16 -qmin 0 -qmax 10 -metadata author="Allen Institute for Neural Dynamics" -movflags +write_colr <output>
```

**Comment by galenlynch** [comment]
output should be a mp4.

**Comment by bruno-f-cruz** [comment]
Thanks I will push the update to the dependencies and document it at the Arch repo.

**Comment by galenlynch** [comment]
Can you remind me  where the arch repo is?

Also where's the video acquisition workflow that you're suggesting people use?

**Comment by bruno-f-cruz** [comment]
Closed via #106
